pkg/factory/risingwave_object_factory.go

```diff
@@ -1099,7 +1111,7 @@ func basicSetupContainer(container *corev1.Container, template *risingwavev1alph
 	container.Resources = template.Resources
 	container.StartupProbe = nil
 	container.LivenessProbe = &corev1.Probe{
-		InitialDelaySeconds: 2,
+		InitialDelaySeconds: 10,
```

Comment:
Out of curiosity: what's the reason to increase `InitialDelaySeconds`?

Reply:
Sometimes, the pods might not be listening on the ports until after some time, this change prevents premature restart due to that. However, this is just a guess on my part, if @arkbriar thinks it's not necessary, then we can remove it!

Reply:
Yep it is, but it also increases the time before the Pod gets ready. Increasing the delay only benefits in the creation scenario but make the scaling slower. I think we should keep it as 2s.